marlowe-dashboard-client/src/Play/View.purs

```diff
@@ -39,7 +39,7 @@ renderPlayState state =
 renderHeader :: forall p. PubKey -> Boolean -> HTML p Action
 renderHeader walletNickname menuOpen =
   header
-    [ classNames $ [ "relative", "flex", "justify-between", "items-center", "leading-none", "border-b", "border-gray", "py-3", "md:py-1", "px-4", "md:px-5pc" ] <> applyWhen menuOpen [ "border-0", "bg-black", "text-white" ] ]
+    [ classNames $ [ "relative", "flex", "justify-between", "items-center", "leading-none", "py-3", "md:py-1", "px-4", "md:px-5pc" ] <> if menuOpen then [ "bg-black", "text-white" ] else [ "border-b", "border-gray" ] ]
```

Comment:
There's a `toggleWhen` utility in the `Css` module for this.

Reply:
I know, but it's just an `if` expression, and I think that one is clearer in intent (as I don't have to be remembering which one was the true/false array)